### ISSUE TITLE
Better Description of the Roles to be Copied is needed.

### ISSUE THREAD

**Issue by ComputerHabit** [MicrosoftDocs/microsoft-365-docs#4115]
The article makes mention of creating new roles for eDiscovery Manager and Investigator, naming them after the fictitious companies.   It does not clarify if you are to make a copy of the existing eDiscovery Managers role and call it Investigator or if you are to copy the Investigator Role.  This role has additional permissions the Managers role does not.

Also if the instruction is to copy the eDiscovery Manager role it does not mention what permissions should be removed for an Investigator.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a94e0469-2c5c-2063-8c3e-7722ceb53577
* Version Independent ID: e99eb8aa-b3b7-8791-aba4-998e209694b5
* Content: [Assign eDiscovery permissions in the Security & Compliance Center - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/assign-ediscovery-permissions?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/assign-ediscovery-permissions.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/assign-ediscovery-permissions.md)
* Service: **o365-seccomp**
* GitHub Login: @markjjo
* Microsoft Alias: **markjjo**

**Comment by markjjo** [comment]
@ComputerHabit I assume that you are referring to [this guidance](https://docs.microsoft.com/en-us/microsoft-365/compliance/set-up-compliance-boundaries?view=o365-worldwide#step-3-create-a-role-group-for-each-agency) from the article able compliance boundaries: "We recommend that you create a role group by copying the built-in eDiscovery Managers group, adding the appropriate members, and removing roles that may not be applicable to your needs."

The guidance is about creating separate role groups for different job roles/responsibilities in different agencies so that you can add different members to those role groups and also use those role groups to assign permissions filters. The names of "eDiscovery Managers" and "eDiscovery Investigators" are just examples in the scenario. The "Investigators" name in the scenario  has no relationship to the  the "Data Investigator" built in role group (if that's what you are referring to). 

We recommend creating new role groups by copying the builtin eDiscovery Manager is easier because then roles are already assigned to them. You can also create new role groups from scratch and then add roles to them.



But what roles you assign to the new role groups that you create is up to you and your organizational needs. It depends on what tasks you want the members of the role groups to perform.

**Comment by ComputerHabit** [comment]
Yes, it's just as an example it leaves a few questions out there.  I had to come to some conclusions on the matter that a tiny bit of clarification might have saved time.  As mentioned I had to review the permissions involved to determine that what I first understood wasn't what was meant.  Then come to a conclusion the additional group was just a way to show you could limit permissions if desired.

I often take examples as literal "how to's".  Having an answer to why I'd need an Investigator vs a Manager is what prompted the questions.
________________________________

**Comment by markjjo** [comment]
@ComputerHabit Thank you for your time and clarification. I went ahead and added a little more detail in the Contoso scenario that tries to clarify. For example, I added that role groups can also be used to control what tasks members can perform (defined by the roles assigned to the role group). I also added to the scenario that investigators should not be able to place holds or export content from a case. That is accomplished by removing the Hold and Export roles from the investigator roles groups. 

Hope this helps. These changes should be published later today or by tomorrow morning.
